hieradata/cluster/chango.yaml

```diff
@@ -0,0 +1,17 @@
+---
+clustershell::groupmembers:
+  chango: {group: "chango", member: "chango[01-03]"}
+profile::core::rke::version: "1.4.6-rc4"
+nm::connections:
```

Comment:
In the scenario where there is only a single interface and its using dhcp, no network configuration is needed as NM defaults to being an dhcp client on active links.  The best configuration is usually no-configuration.

Reply:
Network config was removed as advised.